### DISCUSSION TITLE
page/forum

### DISCUSSION THREAD

**Discussion by giscus** [wangrongding/wangrongding.github.io#8] [Announcements]
# page/forum

荣顶 - 个人博客

http://localhost:12345/page/forum.html

**Comment by wangrongding** [comment]
简单不先于复杂，而是在复杂之后。

**Reply by ChicSparrow** [reply]
智慧从复杂中诞生，在简单中绽放。🌸

**Comment by cnhjp** [comment]
人们表达对其他人的感激之情，有一种方式就是创造一些美好的东西。在你精心制作这个东西的过程中，你会把爱包含在里面，不知不觉传递出去。
